src/canisters/frontend/ic-asset/src/asset/config.rs

```diff
@@ -348,7 +337,7 @@ mod rule_utils {
         headers: Maybe<HeadersConfig>,
         ignore: Option<bool>,
         enable_aliasing: Option<bool>,
-        #[serde(default = "super::default_raw_access")]
+        #[derivative(Default(value = "Some(true)"))]
```

Comment:
An idea as to the underlying cause: this _removes_ the serde default, replacing it with an "impl Default" default.

What would you think about removing the `#[derivative(Default(value = "Some(true)"))]` as well?  At this level, we're interested in knowing what's in the config file rule, and None seems more accurate.